README.md

```diff
@@ -16,8 +16,6 @@ here [Aerospike REST Gateway API Documentation](https://docs.aerospike.com/apido
 ## Getting Started blog posts
 
 1. [Aerospike REST Gateway Introduction](https://medium.com/aerospike-developer-blog/aerospike-rest-client-cb7e5967f423?source=friends_link&sk=0d6d69703e8a77da13ec0c6c012d1c29)
-2. [Dealing with Predicate Expression Filters in Aerospike REST Gateway (Part 1)](https://medium.com/aerospike-developer-blog/dealing-with-predicate-expression-filters-in-aerospike-rest-client-part-1-a43e43ac8c7d?source=friends_link&sk=bc0ed64110578ff6f4804753ca6369da)
-3. [Dealing with Predicate Expression Filters in Aerospike REST Gateway (Part 2)](https://medium.com/aerospike-developer-blog/dealing-with-predicate-expression-filters-in-aerospike-rest-client-part-2-b9d9358c8a4e?source=friends_link&sk=35c37b035d12789aae6272704ef95829)
```

Comment:
Why are we deleting these two items?

Reply:
Yes, I felt they were not for someone who is just getting started with the aerospike gateway